lib/mws/api/base.rb

```diff
@@ -43,19 +44,40 @@ def build_query_from_params(action, params)
 
       def method_missing(name, *args)
         if self.class::ACTIONS.include?(name)
+          warn "[DEPRECATION] `#{name}` will be removed in next major version. " \
+               "Please use `#{name}!` instead."
```

Comment:
Basically we're not going to get rid of methods without bang, but will change their behavior to the same as ones with bang – they'll be raising exceptions.